python/tk_multi_publish2/dialog.py

```diff
@@ -1184,6 +1196,12 @@ def _on_item_context_change(self, context):
     def _on_browse(self, folders=False):
         """Opens a file dialog to browse to files for publishing."""
 
+        # Redundant with disabling UI controls but short circuiting this method
```

Comment:
I think we should hide the browse button at the bottom of the UI as well.

Reply:
Yes, I was originally not testing Publish2 in a DCC, I missed that. It is now not visible.